stl/inc/xstring

```diff
@@ -4268,8 +4277,7 @@ void swap(basic_string<_Elem, _Traits, _Alloc>& _Left,
 template <class _Elem, class _Traits, class _Alloc>
 _NODISCARD basic_string<_Elem, _Traits, _Alloc> operator+(
     const basic_string<_Elem, _Traits, _Alloc>& _Left, const basic_string<_Elem, _Traits, _Alloc>& _Right) {
-    // return string + string
-    basic_string<_Elem, _Traits, _Alloc> _Ans;
+    basic_string<_Elem, _Traits, _Alloc> _Ans(_SOCCC_allocator_arg(), _Left._Getal());

```

Comment:
In new code, we've been trying to use `{}` to construct temporary objects, since it doesn't look like a function call. So, I recommend changing this and the following lines to say `_SOCCC_allocator_arg{}`.